source/blog/2018-12-07-the-ember-times-issue-76.md

```diff
@@ -14,7 +14,7 @@ Join with your **contributions** for **DecEmber** 🐹🎄, read more about usin
 
 ## [Tracked Properties RFC is up for review! 👣](https://github.com/emberjs/rfcs/pull/410)
 
-The [Tracked Properties RFC](https://github.com/tomdale/rfcs/blob/5710d6e6fdc4ae283af7259800beb18aecb95c0d/text/0000-tracked-properties.md) introduces a simpler and more ergonomic system for tracking state change in Ember applications.
+The [Tracked Properties RFC](https://github.com/emberjs/rfcs/blob/be351b059f08ac0fe709bc7697860d5064717a7f/text/0000-tracked-properties.md) introduces a simpler and more ergonomic system for tracking state change in Ember applications.
```

Comment:
:+1:

Reply:
I also commented on https://github.com/emberjs/rfcs/pull/410 so it can be fixed.